codelabs/solace-js-mqtt-pg/index.html

```diff
@@ -249,10 +249,13 @@ <h2 is-upgraded>Code Access</h2>
 <p>That&#39;s it for setup. When you ran the previous two commands, Prisma2 automatically generated data models based on your database&#39;s existing tables and installed a custom Prisma2 client package inside your node_modules that includes all the methods related to CRUD operations on your objects. Don&#39;t run <code>npm i @prisma/client</code>, the generate step does this for you. This easy-to-use client plus being able to store declarative data models in version control (schema.prisma file) is why people love Prisma.</p>
 <p>Now all that&#39;s left is to uncomment the client code that takes User Registration Events and creates an item in your database using Prisma.</p>
 <p><strong>Here are the steps:</strong></p>
-<ul>
-<li>Navigate to <code>mqtt-consumer/src/index.js</code> and uncomment Lines 18 and 41-49, delete or comment out Line 52.</li>
-<li>Navigate to <code>mqtt-consumer/src/event-handlers.js</code> and uncomment Lines 20-27.</li>
-</ul>
+<p>Navigate to <code>mqtt-consumer/src/index.js</code></p>
+<ol type="1">
+<li>uncomment <code>import { PrismaClient } from &#34;@prisma/client&#34;;</code> ,</li>
+<li>uncomment the code block after <code>//initialize and connect prisma client</code></li>
+<li>comment out <code>let prismaClient = null;</code></li>
+</ol>
+<p>Navigate to <code>mqtt-consumer/src/event-handlers.js</code> and uncomment the code block after <code>// do some business logic</code>.</p>
```

Comment:
Note on this - you will have to change it in the `.md` file instead of directly modifying the exported html.. the html file is auto generated from the claat exporting your .md file